io.github.gameflorist.dunedynasty.yaml

```diff
@@ -35,7 +35,7 @@ modules:
       - desktop-file-install -m 644 --dir=$FLATPAK_DEST/share/applications/ io.github.gameflorist.dunedynasty.desktop
 
   - name: allegro5
-    buildsystem: cmake
+    buildsystem: cmake-ninja
```

Comment:
recommended to use cmake-ninja, and not plain cmake, at least in local seem to be working.
https://docs.flathub.org/docs/for-app-authors/linter#module-module_name-buildsystem-is-plain-cmake